internal/client/opendota/match.go

```diff
@@ -8,31 +8,40 @@ import (
 	"net/http"
 )
 
+const (
+	DefaultBaseURL = "https://api.opendota.com/api"
+)
+
 var (
 	ErrFetchMatchDetail = fmt.Errorf("failed to fetch match detail")
 )
 
-type OpenDotaAPI struct {
+type API struct {
```

Comment:
we can then import this as `opendota.API` (since this is an API client for odota, maybe `Client` is also acceptable) instead of `opendota.OpenDotaAPI`

Reply:
I see, yuup2 i agree with this refactoring